python/shotgun_model/shotgun_model.py

```diff
@@ -384,7 +384,7 @@ def reset(self):
 
     def _load_data(
         self, entity_type, filters, hierarchy, fields, order=None, seed=None, limit=None,
-        columns=None, additional_filter_presets=None
+        columns=None, editable_columns=None, additional_filter_presets=None
```

Comment:
what if someone has derived from `ShotgunModel` and subclassed `_load_data` - I think this could potentially break things because you are inserting a new parameter rather than appending it.